ecorest/main.go

```diff
@@ -8,11 +8,12 @@ import (
 )
 
 type restManager struct {
-	ITreeCodesGET      (func() *endpoints.ITreeCodes)
-	EcoGET             (func(url.Values) (*endpoints.BenefitsWrapper, error))
-	EcoSummaryPOST     (func(*endpoints.SummaryPostData) (*endpoints.BenefitsWrapper, error))
-	EcoScenarioPOST    (func(*endpoints.ScenarioPostData) (*endpoints.Scenario, error))
-	InvalidateCacheGET (func())
+	ITreeCodesGET       (func() *endpoints.ITreeCodes)
+	EcoGET              (func(url.Values) (*endpoints.BenefitsWrapper, error))
+	EcoSummaryPOST      (func(*endpoints.SummaryPostData) (*endpoints.BenefitsWrapper, error))
+	EcoFullBenefitsPOST (func(*endpoints.FullBenefitsPostData) (*endpoints.FullBenefitsWrapper, error))
```

Comment:
I think `EcoListPOST` would be more consistent with `EcoSummaryPOST`, as one provides a list-level result and the other an aggregated result. I think word `Full` would hint more to criteria for inclusion and not output format.